static/css/external-site.less

```diff
@@ -0,0 +1,46 @@
+/* ==========================================================================
+   cfgov-refresh
+   Third-Party links pages at /external-site/.
+   ========================================================================== */
+
+.external-site_container {
+    .content_bar {
+      border-top: 1px solid @gray-50;
+      height: 0;
+    }
+
+    .padded-container {
+       background-color: @white;
+    }
+
+    .webfont-regular();
+}
+
+.external-site_text {
+    .webfont-medium();
+    font-style: italic;
+}
+
+.external-site_back-btn {
+    &:link,
+    &:visited {
+      background-color: @darkgray;
+    }
+
+    &:hover {
+      background-color: lighten(@darkgray, 10%);
+    }
+}
+
+.external-site_back-btn, .external-site_proceed-btn {
+    min-width: 180px;
+    text-align: center;
+    margin: 0
```

Comment:
Should follow [playbook property order](https://github.com/cfpb/front-end/blob/master/css.md#property-order)